src/components/PlaylistCard/PlaylistCard.jsx

```diff
@@ -0,0 +1,84 @@
+import { useNavigate } from "react-router-dom";
+import CancelScheduleSendIcon from "@mui/icons-material/CancelScheduleSend";
+import DeleteIcon from "@mui/icons-material/Delete";
```

Comment:
you can use barrel import of mui imports, there'll be a lot of them in places

Reply:
yes will refactor this at the end